docs/scenarios/resolve_structre_ids.md

```diff
@@ -0,0 +1,33 @@
+## Resolving Upwell Structure IDs
+
+A common scenario is to resolve a structure ID into a name. Use the authenticated `/universe/structures/{structure_id}/` endpoint for that. It requires a token that grants the `esi-universe.read_structures.v1` scope on a character that has docking access to the structure. 
+
+The endpoint returns the name of the structure, as well as some additional information such as its system id, type id and the owner corporation id.
+
+## Discovering Upwell Structures
+
+There is currently no unified way discver accessible Upwell structures through ESI. The following methods can be used to discover Upwell structure IDs.
+
+### Public Structures
+
+The public `/universe/structures/` endpoint will return a flat list of IDs containing all fully public structures. *Fully public* in this context means a completely open Access Control List. If the ACL prevents anyone from docking the structure won't show up in this list.
+
+This method has the advantage of not requiring any authentication but is limited in that it only resolves completely public structures. Additionally, most endpoints that consume structure IDs require authentication anyways, so you likely won't be possible to avoid supplying a token anyways.
+
+### Corporation Structures
+
+The authenticated `/corporations/{corporation_id}/structures/` endpoit returns all structures owned by a particular corporation. It requires a token that grants the `esi-corporations.read_structures.v1` scope on a character with Director roles in the corporation.
+
+This endpoint provides a lot of information about the structure, including what services are installed and active, when the fuel expires, what state it currently is in etc. Be aware that it currently *does not* provide the name of the structure.
```

Comment:
Are we sure about this? I can swear Zoetrope added the name to this endpoint?

Reply:
I had some vague memory of that too, but checking the spec shows that there is still no name included in that result.